pkg/config/config.go

```diff
@@ -697,8 +697,17 @@ func InitConfig(config Config) {
 	config.SetKnown("apm_config.replace_tags")
 	config.SetKnown("apm_config.obfuscation.elasticsearch.enabled")
 	config.SetKnown("apm_config.obfuscation.elasticsearch.keep_values")
+	config.SetKnown("apm_config.obfuscation.elasticsearch.transform_values")
+	config.SetKnown("apm_config.obfuscation.elasticsearch.transformer_type")
```

Comment:
I think you forgot this here by accident?

Reply:
Yes, removed.